gradle/java_no_deps.gradle

```
@@ -291,17 +291,26 @@ ext.getJavaLauncherFor = (javaVersionInteger) -> {
 def isJavaVersionAllowedForProperty(JavaVersion version, String propertyPrefix = "") {
   def minProp = propertyPrefix.isEmpty() ? 'minJavaVersionForTests' : "${propertyPrefix}MinJavaVersionForTests"
   def maxProp = propertyPrefix.isEmpty() ? 'maxJavaVersionForTests' : "${propertyPrefix}MaxJavaVersionForTests"
-  if (project.hasProperty(minProp) && project.getProperty(minProp).compareTo(version) > 0) {
+  def definedMin = project.hasProperty(minProp)
+  def definedMax = project.hasProperty(maxProp)
+  if (definedMin && project.getProperty(minProp).compareTo(version) > 0) {
     return false
   }
-  if (project.hasProperty(maxProp) && project.getProperty(maxProp).compareTo(version) < 0) {
+  //default to the general min if defined and specific one is was not defined
+  if (!propertyPrefix.isEmpty() && !definedMin && project.hasProperty('minJavaVersionForTests') && project.getProperty('minJavaVersionForTests').compareTo(version) > 0) {
+    return false
+  }
+  if (definedMax && project.getProperty(maxProp).compareTo(version) < 0) {
+    return false
+  }
+  if (!propertyPrefix.isEmpty() && !definedMax && project.hasProperty('maxJavaVersionFollrTests') && project.getProperty('maxJavaVersionForTests').compareTo(version) < 0) {
```

typo I think right ?
```suggestion
  if (!propertyPrefix.isEmpty() && !definedMax && project.hasProperty('maxJavaVersionForTests') && project.getProperty('maxJavaVersionForTests').compareTo(version) < 0) {
```

---

I should have typed `ll` but not in the shell :D